setup.py

```diff
@@ -296,7 +306,7 @@ def run(self):
 try:
     import pypandoc
     long_description = pypandoc.convert("README.md", "rst")
-except (IOError, ImportError):
+except:
```

Comment:
Thanks, I agree that this would suffice in this case, but really regardless of the reason why this fails we wouldn't like it to be a fatal error that prevents install, as we have a reasonable fallback option (don't add a long description). So I think I'll leave it as-is.